requirements/base.txt

```diff
@@ -4,7 +4,8 @@ Django==1.8.17
 dj.choices==0.10.0
 django-extensions==1.5.9
 django-filter==0.13.0
-django-import-export==0.4.2
+# bump to newer when https://github.com/django-import-export/django-import-export/pull/559 will be merged
```

Comment:
Would you please mind checking the comments in https://github.com/django-import-export/django-import-export/pull/559 to see if we can bump the version now?